web/httputils/httputils.go

```diff
@@ -23,3 +27,46 @@ func GetQueryParams(r *http.Request) url.Values {
 	r.ParseForm()
 	return r.Form
 }
+
+type OutputFormat int
+
+const (
+	OutputText = 1
```

Comment:
The text output format is already gone in master (at least in the web API) - see https://github.com/prometheus/prometheus/blob/master/web/api/query.go.